crates/metadata/src/utils.rs

```diff
@@ -58,11 +58,26 @@ where
     deserializer.deserialize_str(Visitor)
 }
 
-/// Strips a single whitespace at the start and removes trailing spaces
-pub fn trim_extra_whitespace(item: &str) -> &str {
```

Comment:
This might be able to be restored and remove the duplicate impls, see usage site above with `AsRef<[u8]>` bound

Reply:
Not even needed, was able to nail it using just str refs :ok_hand: